### DISCUSSION TITLE
one dimensional problems

### DISCUSSION THREAD

**Discussion by arunkv7** [FourierFlows/GeophysicalFlows.jl#310] [General]
I would like to know if implementing a numerical pseudospectral solver for one dimensional equation such as Burgers or Nonlinear Schrödinger equation is straight forward as done here with FourierFlows in Julia

**Comment by arunkv7** [comment]
I am sorry I missed taking a look at the one dimensional shallow water example provided with FourierFlows.jl 

Thanks

**Reply by navidcy** [reply]
Let us know if you have difficulty.
